js/translation-helpers.js

```diff
@@ -63,6 +64,15 @@ $gp.translation_helpers = (
 				$tab.addClass('current');
 				$("#"+tab_id).addClass('current');
 			},
+			reply_comment_form: function( $comment ) {
+				var commentId = $comment.attr('data-commentid') ;
+				$('#comment-reply-' + commentId).toggle();
+				if ('Reply' === $comment.text()) {
+					$comment.text('Cancel Reply');
+				} else {
+					$comment.text('Reply');
```

Comment:
Some small whitespace issues:

```suggestion
				var commentId = $comment.attr( 'data-commentid' );
				$( '#comment-reply-' + commentId ).toggle();
				if ( 'Reply' === $comment.text() ) {
					$comment.text( 'Cancel Reply' );
				} else {
					$comment.text( 'Reply' );
```